prometheus_client/django/exposition.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from django.http import HttpResponse
+from django.views import View
+
+import prometheus_client
+from prometheus_client import multiprocess
+from prometheus_client.exposition import _bake_output
+from tests.test_exposition import registry
```

Comment:
```suggestion
from prometheus_client.registry import registry
```

I think this is what you meant and should also fix the lint errors!